TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/snapshot.py

```diff
@@ -23,6 +23,7 @@
                 'schema': {
```

Comment:
Ideally there would be two tests for this:
- one on the python side (if `role` is in the services.yaml, it should be included in the call to the java snapshot tool, a la [this test](https://github.com/opensearch-project/opensearch-migrations/blob/main/TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_snapshot.py#L290) and the ones nearby)
- and one on the CDK side, where we set the `managedServiceSourceSnapshotEnabled` and make sure the policy gets created / added to the YAML (kind of like [this one](https://github.com/opensearch-project/opensearch-migrations/blob/main/deployment/cdk/opensearch-service-migration/test/reindex-from-snapshot-stack.test.ts#L311).

The python test is a low lift with decent helpful value to effectively document how role should effect the rest of the system. The CDK test is a much higher lift that should have been done in my previous PR, but wasn't. And is potentially going to be replaced in the near-ish future.

So I'd really like the python test, the CDK test is optional.